test/app-luatest/interval_test.lua

```diff
@@ -34,3 +34,8 @@ end
 g.test_check_interval_encode_ffi = function()
     t.assert_equals(bin, msgpackffi.encode(val))
 end
+
+g.test_is_interval = function()
+    t.assert(itv.is_interval(val))
+    t.assert_not(itv.is_interval(val:totable()))
```

Comment:
I would remove the second assertion or replace it with empty table: `t.assert_not(itv.is_interval({}))`.
Feel free to ignore.

Reply:
Thanks, fixed!